Makefile

```diff
@@ -9,7 +9,7 @@ BUILD_DATE := $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 
 
 .PHONY: all
-all: clean test build
+all: clean test build sanity-check
```

Comment:
Do we need this in `make all`? I'm okay with adding this addition make target, but I question the need to run it by default.

Reply:
I will remove it from `make all`.